argilla-server/src/argilla_server/api/handlers/v1/workspaces.py

```diff
@@ -88,6 +105,10 @@ async def list_workspaces_me(
     else:
         workspaces = await accounts.list_workspaces_by_user_id(db, current_user.id)
 
+    for workspace in workspaces:
+        await telemetry_client.track_crud_workspace(action="read", workspace=workspace)
+    await telemetry_client.track_crud_workspace(action="read", workspace=None, count=len(workspaces))
```

Comment:
@frascuchon It is to keep track of list like operations but not differentiate too much in the naming CRUD. If you think it is interesting for development, I will create a separate "list" action. Otherwise I will leave it.

Reply:
I reviewed the code and see I did keep track of the list-like thing in other places too.